static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -67,8 +67,7 @@ export type DashboardWidgetModalOptions = {
   defaultTableColumns?: readonly string[];
   defaultTitle?: string;
   displayType?: DisplayType;
-  fromDiscover?: boolean;
-  fromLibrary?: boolean;
+  source: 'discoverv2' | 'dashboards' | 'library' | 'issueDetail';
```

Comment:
Just a nit, if we could store these as constants I think that would be an improvement!

Reply:
@edwardgou-sentry sure, I'll convert it